meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -63,7 +63,7 @@ MENDER_MTD_UBI_DEVICE_NAME ??= "${MENDER_MTD_UBI_DEVICE_NAME_DEFAULT}"
 MENDER_MTD_UBI_DEVICE_NAME_DEFAULT = ""
 
 # Filesystem type of data partition. Used for both FS generation and fstab construction
-# Leave as default (auto) to generate a partition using the same Filesystem as 
```

Comment:
Nothing major, but in general this will dirty the blame-log.

Reply:
My editor does this, and I was hoping that this would  slip trough :)

Reply:
Reason I'm saying it is because I'm getting yelled at for this also. The solution I found was to add a git-hook instead. So that only the staged git diff will be whitespace-filtered ;)

Gonna let it slide for now though :p

Reply:
I have updated :)